examples/kubernetes/templates/v1/cilium-cm.yaml

```diff
@@ -70,6 +70,21 @@ data:
   ct-global-max-entries-tcp: "524288"
   ct-global-max-entries-other: "262144"
 
+  # Pre-allocation of map entries allows per-packet latency to be reduced, at
+  # the expense of up-front memory allocation for the entries in the maps. The
+  # default below value will minimize memory usage in the default installation;
+  # users who are sensitive to latency may consider enabling this option.
+  #
+  # If this value is modified, then during the next Cilium startup the restore
+  # of existing endpoints and tracking of ongoing connections may be disrupted.
+  # This may lead to policy drops or a change in loadbalancing decisions for a
+  # connection for some time. Endpoints may need to be recreated to restore
+  # connectivity.
+  #
+  # For users upgrading from Cilium 1.3 or earlier, to minimize disruption
```

Comment:
Let's rephrase this to say that Cilium <= 1.3 had this value to be always true so setting it to false and upgrading from <= 1.3 to >= 1.4 will cause some one-time disruptions during the upgrade. The way it is written right now, people may mistake it as for causing disruptions continuously.

Let's also specify that for versions < 1.4, this option has no effect. We should do this for other fields as well but doesn't hurt to start with this one.

Reply:
@tgraf I adjusted for this feedback, PTAL.